package.json

```diff
@@ -21,14 +21,16 @@
     "node": ">= 7.6.0"
   },
   "devDependencies": {
-    "babel-cli": "^6.24.0",
-    "babel-core": "^6.24.0",
+    "babel-cli": "^7.0.0-alpha.19",
```

Comment:
`alpha`
![](https://media.giphy.com/media/lKZEeXJGhU1d6/giphy.gif)

Reply:
Yes, for the blog post. I might wait for beta/major before publishing this update.

The babel maintainers are working heavy on it, almost a release weekly. Shouldn't take to long but my measurements prove improvements in the bundle size when using the new babel-preset-env v2 (which requires babel 7)